### PR TITLE
[Tariff Synchronizer] run once per day, reduce extra email notifications about failed updates

### PR DESCRIPTION
**Pull request by rusllonrails**
[TRELLO CARD](https://trello.com/c/RwWAkrwG/2050-tariff17-change-sync-and-email-notifications-for-failed-updates)

*THIS PR COVERS:*

1) [UpdatesSynchronizerWorker] now would be runned once per day at 10PM.

2) [TariffSynchronizer] removed `Mailer.failures_reminder` as we no longer need it

### CODE REVIEW

**Comment by rusllonrails** [comment]
[[TARIFF17] Change sync and email notifications for failed updates](https://trello.com/c/RwWAkrwG/2050-tariff17-change-sync-and-email-notifications-for-failed-updates)

File: lib/tariff_synchronizer/mailer.rb [hunk 1]
```diff
@@ -27,12 +27,6 @@ def failed_download(exception, url)
       mail subject: "#{subject_prefix(:error)} Trade Tariff download failure"
     end
 
-    def failures_reminder(file_names)
```
Comment (alexey-klimuk) [inline lib/tariff_synchronizer/mailer.rb]: @rusllonrails @matthewford maybe we should keep mailer code in case reuse it in future

Comment (rusllonrails) [reply lib/tariff_synchronizer/mailer.rb]: @alexey-klimuk not sure, @matthewford what is your opinion?

Comment (matthewford) [reply lib/tariff_synchronizer/mailer.rb]: that's what git is for, its still here if we need it, can remove.

**Pull request merged**